### PR TITLE
DDPB-3584: Update checklist sync metadata

### PR DESCRIPTION
**Pull request by acsauk-mojd**
## Purpose
Currently, when syncing a checklist with Sirius the user associated with the event in Sirius is a generic API user. This changes existing business processes for OPG that rely on knowing who has submitted a checklist so we should ensure this detail is available in Sirius. 

In order to provide Sirius with the required information we need to ensure we are sending:

* Reporting period years
* The report type (PF, HW or Combined)
* The email address of the user that submitted the checklist

Fixes DDPB-3584

## Checklist
- [x] I have performed a self-review of my own code
- [ ] I have updated documentation (Confluence/ADR/tech debt doc) where relevant
- [x] I have added tests to prove my work
- [ ] The product team have approved these changes

### CODE REVIEW

File: client/tests/phpunit/Pact/SiriusApiGatewayClientTest.php [hunk 1]
```diff
@@ -284,8 +307,14 @@ private function setUpChecklistPdfPactBuilder(string $caseRef, string $reportPdf
             ]);
 
         $this->builder
-            ->uponReceiving('A submitted checklist pdf')
+            ->uponReceiving('A submitted checklist pdf  a')
```
Comment (jamesrwarren) [inline client/tests/phpunit/Pact/SiriusApiGatewayClientTest.php:310]: Is this a typo?

**Review by jamesrwarren — COMMENTED** [review]
Just the one question really. Otherwise it looks good.

File: client/tests/phpunit/Helpers/SiriusHelpers.php [hunk 2]
```diff
@@ -57,16 +57,33 @@ static public function generateSiriusSupportingDocumentUpload(int $submissionId,
...
+        int $year,
+        string $type
+    )
     {
         $file = (new SiriusDocumentFile())
             ->setName($fileName)
             ->setMimetype('application/pdf')
             ->setSource(base64_encode($fileContents));
 
+        $attributes =( new SiriusChecklistPdfDocumentMetadata())
```
Comment (coderste) [inline client/tests/phpunit/Helpers/SiriusHelpers.php:76]: Formatting issue 🙂

**Review by coderste — COMMENTED** [review]
Just the one issue with the spacing 🙂

File: client/tests/phpunit/Pact/SiriusApiGatewayClientTest.php (hunk 1 above)
Comment (acsauk-mojd) [reply client/tests/phpunit/Pact/SiriusApiGatewayClientTest.php:310]: Ah yeah - I was forcing a new check during dev as I'd already had a successful test run so pact wouldn't run a further test with the same name. I'll revert.

File: client/tests/phpunit/Helpers/SiriusHelpers.php (hunk 2 above)
Comment (acsauk-mojd) [reply client/tests/phpunit/Helpers/SiriusHelpers.php]: Good eyes - fixed now!

**Review by jamesrwarren — APPROVED** [review]

**Review by coderste — DISMISSED** [review]
LGTM 👍🏻

**Review by jamesrwarren — APPROVED** [review]

**Pull request merged**